Apps.md

```diff
@@ -0,0 +1,80 @@
+Apps for the app builder are developed as standalone applications which should be able to run with `npm start` in standalone mode.
+
+Each App will be deployed in npm using the `@entando` namespace and will export in their dist folder several items used by the app builder when integrating it
+
+# Exports
+several different items need to be exported by the App so that they can be consumed by the app builder.
+
+Each paragraph reports in angle brackets the name of the object that the App itself need exporting
+
+## ID <id>
+the unique identifier for the app:
+
+i.e. `cms`
+
+## Routing
```

Comment:
is the routing paragraph still needed after the removal of @entando/router?

Reply:
I'll be taking care of this later

Reply:
cool, thanks